app/routes/circulars/circulars.lib.ts

```diff
@@ -76,9 +80,14 @@ const subjectMatchers: SubjectMatcher[] = [
     /Baksan\sNeutrino\sObservatory\sAlert[.\s_-]*(\d{6}.\d{2})/i,
     ([, id]) => `Baksan Neutrino Observatory Alert ${id}`,
   ],
-  [/EP[.\s_-]*(\d{6}[a-z])/i, ([, id]) => `EP${id}`],
-  [/FRB[.\s_-]*(\d{8}[a-z])/i, ([, id]) => `FRB ${id}`.toUpperCase()],
+  [/EP[.\s_-]*(\d{6}[a-z])/i, ([, id]) => `EP${id.toLowerCase()}`],
+  [/SN[.\s_-]*(\d{4}[a-zA-Z]*)/i, ([, id]) => `SN${id.toLowerCase()}`],
```

Comment:
I'm not sure this quite captures it.  The first 26 of the year are capital, then switch to 2-letter lower case, and eventually 3-letter lower case.  Sorry ... astronomy!  Here's a good description: http://astro.vaporia.com/start/sndesignator.html

Reply:
ok, I changed it so that:
- if there is one letter at the end, it should be capital
- if there is more than one letter at the end, they will be lowercase
- there is a space between SN and the id. This was how it was in the documentation you linked above.